setup.py

```diff
@@ -53,6 +54,7 @@
     install_requires=requirements,
     license="MIT license",
     long_description=readme,
+    long_description_content_type='text/markdown',
```

Comment:
```suggestion
    long_description_content_type='text/markdown',
    long_description=readme,
```
i read somewhere that the order is important.